apps/site/assets/static/robots.txt

```diff
@@ -8,3 +8,7 @@ Disallow: /search?*
 Disallow: /*?*preview*&vid=*
 Disallow: /schedules/*/line?*date*
 Disallow: /org/*
+Disallow: /charlie/*
+Disallow: /policies/terms-use-charlie/*
```

Comment:
The urls we wish to not index with this pattern looks like there is no final slash.  I believe it should be `/policies/terms-use-charlie*`
This might need to be double checked with the `robots.txt` documentation.
Also this needs to be updated in the `app.html.eex` file to not include the final slash